.travis.yml

```diff
@@ -222,4 +222,4 @@ matrix:
 before_install:
   - eval "${MATRIX_EVAL}"
 
-script: cmake -DCMAKE_INSTALL_PREFIX=/tmp -DCMAKE_BUILD_TYPE="$BUILD_TYPE" -DSUPPORT_JOURNALD=ON -DSUPPORT_DTLS="$DTLS" -DSUPPORT_ZMQ="$ZMQ" . && make -k && make test && make install
+script: cmake -DCMAKE_INSTALL_PREFIX=/tmp -DCMAKE_BUILD_TYPE="$BUILD_TYPE" -DSUPPORT_JOURNALD=ON -DSUPPORT_DTLS="$DTLS" -DSUPPORT_ZMQ="$ZMQ" . && make -k && make test && make install && (cd src/tests/vermonttest/; ./vermonttest)
```

Comment:
Perhaps use 'make test' instead?

Reply:
```
19:41 $ make test
Running tests...
Test project /home/brownn/work/vermont/build
    Start 1: vermont_cppcheck_test
1/8 Test #1: vermont_cppcheck_test ............   Passed    0.21 sec
    Start 2: core_cppcheck_test
2/8 Test #2: core_cppcheck_test ...............   Passed    0.16 sec
    Start 3: common_cppcheck_test
3/8 Test #3: common_cppcheck_test .............   Passed    3.19 sec
    Start 4: ipfixlolib_cppcheck_test
4/8 Test #4: ipfixlolib_cppcheck_test .........   Passed    0.67 sec
    Start 5: modules_cppcheck_test
5/8 Test #5: modules_cppcheck_test ............   Passed    4.99 sec
    Start 6: vermonttest_build
6/8 Test #6: vermonttest_build ................   Passed    0.18 sec
    Start 7: vermont
7/8 Test #7: vermont ..........................   Passed   11.89 sec
    Start 8: example_2
8/8 Test #8: example_2 ........................   Passed    0.01 sec

100% tests passed, 0 tests failed out of 8

Total Test time (real) =  21.32 sec
```

Reply:
You are completely right. And in fact, it already is. The test named vermont already calls the vermonttest. So this commit is unnecessary. I removed it.